packages/shared/src/extension/chrome/popup/components/settings.tsx

```diff
@@ -9,6 +11,8 @@ import {
   ListItemSecondaryAction,
 } from '@material-ui/core';
 
+import { Label as LabelIcon } from '@material-ui/icons';
```

Comment:
It is better to import icons from specific location to avoid an increase in the bundle size

Reply:
Webpack does tree shaking so it doesn't change anything

Reply:
why the yttrex one is bigger, then?